test/parallel/util-parse-env.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
```

Comment:
What happens when `util.parseEnv` is called with `NODE_OPTIONS`? I'm pretty sure it won't affect the process since NODE_OPTIONS is evaluated before `Environment` creation, but in case we want to pursue this API, this should be documented.

Reply:
parseEnv returns an object and doesn't update the process.env attributes.